### PR TITLE
docs: Correct minor spelling issues

### PR DESCRIPTION
**Pull request by iksrandeb**
Correct spelling issues in the 'extra-build-config.md' file.

Signed-off-by: Andrzej Bednarski <andrzej.bednarski@gmail.com>

**- What I did**
Corrected spelling issues

**- How I did it**
By reading the documentation and fixing

### CODE REVIEW

File: docs/extra-build-config.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ To remove (or adjust) this delay set these variables in local.conf:
 ## Set overclocking options
 
 The Raspberry Pi can be overclocked. As of now overclocking up to the "Turbo
-Mode" is officially supported by the raspbery and does not void warranty. Check
+Mode" is officially supported by the raspberry and does not void warranty. Check
```
Comment (iksrandeb) [inline docs/extra-build-config.md:80]: Since Raspberry is a product and registered name, shall it be capitalized and changed to "Raspberry PI"?

Comment (agherzan) [reply docs/extra-build-config.md:80]: I think that makes sense.

**Review by iksrandeb — COMMENTED** [review]
Capitalized name for raspberry.

**Review by agherzan — APPROVED** [review]
Nice one. Thanks for your contribution.

**Pull request merged**